schema.ts

```diff
@@ -1203,6 +1211,44 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
   modifiedByProperty?: PropertyIdentifier<K>;
 
   // TODO(dweitzman): Only support options in the typing when the codaType is ValueHintType.SelectList.
+
+  // TODO(patrick): Unhide this.
+  /**
+   * This property should be typed as either an Email or a Person.
+   * @hidden
+   */
+  permissionUserProperty?: K;
```

Comment:
I'm not sure if this is possible.  Would it be desirable to have a single `permissionPropery` which is an object?  That way we can change it easier and also not have to prefix every property w/permission.

Reply:
Oh, interesting... so the validation of it would just be dynamic based on what kind of principal the table is marked as? That could work...